R/idr0021 - Segmentation.ipynb

```diff
@@ -0,0 +1,702 @@
+    "\n",
+    "Each user has a partial copy of the dataset ('Lab1' / YOUR USER / Project: 'idr0021'). Some of the images are also part of the dataset 'R-dataset' which you can use for practicing the segmentation and annotation steps."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Exercise:** Go to https://outreach.openmicroscopy.org , log in as your given user, and find the dataset ('Lab1' / YOUR USER / Project: 'R-dataset'). Leave the window open, as you will use OMERO.web to check the results as we're going along."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## How to use this notebook"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The grey blocks with the In\\[ \\] notation to the left are code blocks. Click on a block to select it and click the play button in the toolbar to execute the block of code (alternatively hold SHIFT key and hit RETURN). The notation will change to In\\[*\\] to indicate that this code is currently running. The output is displayed below the block. Wait for it to finish. When the execution is finished it will change to ln\\[1\\] showing that this was the first block of code you ran (ln\\[2\\], ln\\[3\\], ... for the next executed code blocks). \n",
+    "\n",
+    "As we go through the notebook you're expected to run the code blocks in order (unless explicitely mentioned otherwise).\n",
+    "\n",
+    "Here's an example:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "message(\"Just wait for a bit...\")\n",
+    "Sys.sleep(3)\n",
+    "message(\"Done.\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can change the code and run the block again.\n",
+    "\n",
+    "**Exercise:** Change the message which is printed and run it again."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Let's get started..."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Load the necessary [romero-gateway](https://github.com/ome/rOMERO-gateway) and [EBImage](https://bioconductor.org/packages/release/bioc/html/EBImage.html) library. We're currently using romero-gateway 0.4.3, EBImage 4.13.0 with R 3.4.1 . "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "library(romero.gateway)\n",
+    "library(EBImage, warn.conflicts = FALSE)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Log into the OMERO server"
```

Comment:
``Log into`` -> ``Log in to``. 

Maybe also add the explanation about

``Execute this cell first, this will display a prompt into which you will put your credentials.``